### PR TITLE
config,volcli,volmaster: send full policy JSON

### PR DESCRIPTION
**Pull request by unclejack**
This PR changes the REST api for the policy listing to send an array of complete policies.

### CODE REVIEW

File: volmaster/daemon.go [hunk 1]
```diff
@@ -203,7 +203,12 @@ func (d *DaemonConfig) handlePolicyList(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	content, err := json.Marshal(policies)
+	response := []*config.Policy{}
+	for _, policy := range policies {
+		response = append(response, policy)
```
Comment (erikh) [inline volmaster/daemon.go]: why not just marshal policies directly here exactly?

**Comment by erikh** [comment]
LGTM, but I think the policies loop is superfluous, unless there's good reason, please remove it.

File: volmaster/daemon.go (hunk 1 above)
Comment (unclejack) [inline volmaster/daemon.go]: Fair enough, I'll refactor the code and update.

**Comment by unclejack** [comment]
@erikh PTAL

**Comment by dseevr** [comment]
LGTM

**Comment by erikh** [comment]
LGTM

**Pull request merged**